src/assets/strings/LandingResources.js

```diff
@@ -0,0 +1,12 @@
+const LandingResources = {
```

Comment:
This file is literally named LandingResources? Doesn't it make sense to have it in the `components/Landing` folder itself? That way we can avoid having component specific constants outside the component folder

Reply:
I agree. Since we wont be doing any loc related to these files they don't need to be in a common location.